public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx

```diff
@@ -271,6 +276,24 @@ export class GeneralSettingsEditView
             >
               <RadioButtonGroup onChange={model.onTooltipChange} options={GRAPH_TOOLTIP_OPTIONS} value={graphTooltip} />
             </Field>
+
+            <Field
+              label="Preload panels"
+              description="When enabled all panels will start loading as soon as the dashboard has been loaded"
+
+              // TODO
+              // label={t('dashboard-settings.general.panel-options-graph-tooltip-label', 'Graph tooltip')}
+              // description={t(
+              //   'dashboard-settings.general.panel-options-graph-tooltip-description',
+              //   'Controls tooltip and hover highlight behavior across different panels. Reload the dashboard for changes to take effect'
+              // )}
```

Comment:
commented out code can be removed (assume this is a reference for adding the translations)